### ISSUE TITLE
mdmath.style has no effect

### ISSUE THREAD

**Issue by ostrokach** [goessner/mdmath#68]
### Summary

The README states that we can use `mdmath.style` to specify a custom stylesheet. However, I have not been able to get this to work, and in VSCode when I add "mdmath.style" to user settings this keyword is underlined in green with a comment: "Unknown configuration setting".
 
### Expected behavior

I would expect that I could set `"mdmath.style": "https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css"` to restore default VSCode markdown sytle.

### Actual behavior

Setting `"mdmath.style": anything` does not seem to make any difference?

### Steps to reproduce

1. Set `"mdmath.style"` to anything in user settings.

### Environment

```text
     Operating system: Linux Mint
       VSCode version: 1.32.3
Markdown+Math version: 2.3.9 
```

**Comment by flipMWD** [comment]
Yes, `settings.json` file highlights as unknown configuration. You can still use the default style from VSCode by downloading the CSS from the link above, and replacing `vscode-texmath.css` from `C:\Users\User\.vscode\extensions\goessner.mdmath-2.3.9\css` with the `markdown.css` renamed.

**Comment by yuekai** [comment]
This is easy enough to fix on your own. See #77

**Comment by ostrokach** [comment]
Thanks!

It may be easy if you know javascript and vscode, but I still have no clue with what's going on in 6 / 8 lines in that PR :upside_down_face:.

**Comment by goessner** [comment]
This should be fixed now with version 2.4.0

thanks

**Comment by ostrokach** [comment]
Thanks!
